core/src/test/resources/application.conf

```diff
@@ -1,196 +1,12 @@
-webservice {
-  port = 8000
-  interface = 0.0.0.0
-  instance.name = "reference"
-}
-
-akka {
-  loggers = ["akka.event.slf4j.Slf4jLogger"]
-  actor {
-    default-dispatcher {
-      fork-join-executor {
-        # Number of threads = min(parallelism-factor * cpus, parallelism-max)
-        # Below are the default values set by Akka, uncomment to tune these
-
-        #parallelism-factor = 3.0
-        #parallelism-max = 64
-      }
-    }
-  }
-}
-
-spray.can {
-  server {
-    request-timeout = 40s
-  }
-  client {
-    request-timeout = 40s
-    connecting-timeout = 40s
-  }
-}
-
-system {
-  // Use the new route of execution starting ShadowWorkflowActor
-  // Once all features related to pluggable backends are in place, this will be deprecated
-  shadowExecutionEnabled = false
-
-  // If 'true', a SIGINT will trigger Cromwell to attempt to abort all currently running jobs before exiting
-  abort-jobs-on-terminate = false
-}
+// Additions/updates to main's application.conf
```

Comment:
If they aren't, then I believe the test would have failed already, as the main conf would have overwritten the test conf values with original, wrong, values.